tests/safeds/ml/classical/classification/test_classifier.py

```diff
@@ -284,3 +284,104 @@ def test_should_return_1_if_never_expected_to_be_positive(self) -> None:
     def test_should_raise_if_table_is_not_tagged(self, table: Table) -> None:
         with pytest.raises(UntaggedTableError):
             DummyClassifier().precision(table)  # type: ignore[arg-type]
+
+
+class TestRecall:
+    def test_should_compare_result(self) -> None:
+        table = Table.from_dict(
+            {
+                "predicted": [1, 1, 0, 2],
+                "expected": [1, 0, 1, 2],
+            },
+        ).tag_columns(target_name="expected")
+
+        assert DummyClassifier().recall(table, 1) == 0.5
+
+    def test_should_compare_result_with_different_types(self) -> None:
+        table = Table.from_dict(
+            {
+                "predicted": [1, "1", "0", "2"],
+                "expected": [1, 0, 1, 2],
+            },
+        ).tag_columns(target_name="expected")
+
+        assert DummyClassifier().recall(table, 1) == 0.5
+
+    def test_should_return_1_if_never_expected_to_be_positive(self) -> None:
+        table = Table.from_dict(
+            {
+                "predicted": ["lol", "1", "0", "2"],
+                "expected": [1, 0, 1, 2],
+            },
+        ).tag_columns(target_name="expected")
+
+        assert DummyClassifier().recall(table, 1) == 1.0
+
+    @pytest.mark.parametrize(
+        "table",
+        [
+            Table.from_dict(
+                {
+                    "a": [1.0, 0.0, 0.0, 0.0],
+                    "b": [0.0, 1.0, 1.0, 0.0],
+                    "c": [0.0, 0.0, 0.0, 1.0],
+                },
+            ),
+        ],
+        ids=["untagged_table"],
+    )
+    def test_should_raise_if_table_is_not_tagged(self, table: Table) -> None:
+        with pytest.raises(UntaggedTableError):
+            DummyClassifier().recall(table)  # type: ignore[arg-type]
+
+class TestF1_score:
+    def test_should_compare_result(self) -> None:
+        table = Table.from_dict(
+            {
+                "predicted": [1, 1, 0, 2],
+                "expected": [1, 0, 1, 2],
+            },
+        ).tag_columns(target_name="expected")
+
+        f1 = DummyClassifier().f1_score(table, 1)
+        assert f1 <= 0.667
+        assert f1 > 0.666
```

Comment:
Have a look at the [pytest.approx](https://docs.pytest.org/en/latest/reference/reference.html#pytest.approx) function for the comparison of floats.